docs/gallery_scripts_template/plot_spyogenes_subplots_ms_matplotlib.py

```
@@ -1,16 +1,21 @@
+====================================================
 
 Here we show how we can plot multiple chromatograms across runs together
 """
 
 import pandas as pd
 import requests
 import zipfile
-import numpy as np
 import matplotlib.pyplot as plt
+import sys
+import sys
```

Ruff (0.8.2)

Redefinition of unused `sys` from line 12

Remove definition: `sys`

(F811)